sixtools/AD_Handler.py

```diff
@@ -0,0 +1,107 @@
+from databroker.assets.handlers_base import HandlerBase
+import os.path
+import pandas as pd
+import numpy as np
+
+
+class HDF5SingleHandler_centroid7xn(HandlerBase):
```

Comment:
This could become `HDF5SingleHandlerDataFrame(HandlerBase):`

This then becomes much more generic

Reply:
Agreed I like this idea as it means that only the small wrapper function is bespoke to the current situation. I will make this change on Monday morning.